<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_empresa_datos_actualizar</class>
 <widget class="QDialog" name="form_empresa_datos_actualizar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>555</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>239</red>
        <green>235</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>231</red>
        <green>231</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>103</red>
        <green>103</green>
        <blue>103</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>138</red>
        <green>138</green>
        <blue>138</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>239</red>
        <green>235</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>239</red>
        <green>235</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>231</red>
        <green>231</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>239</red>
        <green>235</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>231</red>
        <green>231</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>103</red>
        <green>103</green>
        <blue>103</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>138</red>
        <green>138</green>
        <blue>138</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>239</red>
        <green>235</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>239</red>
        <green>235</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>231</red>
        <green>231</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>103</red>
        <green>103</green>
        <blue>103</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>239</red>
        <green>235</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>231</red>
        <green>231</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>103</red>
        <green>103</green>
        <blue>103</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>138</red>
        <green>138</green>
        <blue>138</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>103</red>
        <green>103</green>
        <blue>103</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>103</red>
        <green>103</green>
        <blue>103</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>239</red>
        <green>235</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>239</red>
        <green>235</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>207</red>
        <green>207</green>
        <blue>207</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Datos empresa actualizar</string>
  </property>
  <property name="styleSheet">
   <string notr="true">selection-background-color: rgb(255, 170, 127);
background-color: rgb(239, 235, 231);</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>611</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(172, 55, 26);
font:  11pt &quot;KacstOne&quot;;</string>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="icon">
     <iconset>
      <normaloff>Íconos/boton-actualizar.png</normaloff>Íconos/boton-actualizar.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Actualizar Empresa</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>571</width>
       <height>411</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(239, 235, 231);
font:  10pt &quot;KacstOne&quot;;
QLabel{
background-color: rgb(239, 235, 231);
}

QLineEdit{
background-color: rgb(255, 255, 255);
}

QTextEdit{
background-color: rgb(255, 255, 255);
}

QComboBox{
background-color: rgb(255, 255, 255);
}

QDateEdit{
background-color: rgb(255, 255, 255);
}

</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset>
        <normaloff>Íconos/tarjeta-personal-de-datos-de-contacto.png</normaloff>Íconos/tarjeta-personal-de-datos-de-contacto.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Actualizar Datos</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>24</x>
         <y>16</y>
         <width>521</width>
         <height>341</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox{
background-color: rgb(239, 235, 231);
border-style: solid;
border-width: 5px;
border-color: rgb(172, 55, 26);
}

QLabel{
background-color: rgb(239, 235, 231);
}

QLineEdit{
background-color: rgb(255, 255, 255);
}

QTextEdit{
background-color: rgb(255, 255, 255);
}

QComboBox{
background-color: rgb(255, 255, 255);
}

QDateEdit{
background-color: rgb(255, 255, 255);
}

</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLineEdit" name="lne_ciudad">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>60</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="lne_ingresos_brutos">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>240</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="lne_website">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>180</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="lne_telefono">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>120</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_48">
        <property name="geometry">
         <rect>
          <x>24</x>
          <y>300</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nombre de  fantasia:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lne_email">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>150</y>
          <width>261</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_40">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Ciudad:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_47">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>270</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>N° de CUIT:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_42">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>120</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Teléfono/Fax:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_39">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nombre de la Empresa:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lne_direccion">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>90</y>
          <width>300</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="lne_nombre">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>30</y>
          <width>301</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>KacstOne</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_43">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>150</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>E-mail:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lne_cuit">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>270</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QComboBox" name="cbx_tipo_iva">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>210</y>
          <width>161</width>
          <height>23</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Responsable inscripto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Exento</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Monotributo</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_41">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Dirección:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_45">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>210</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Tipo de IVA:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_44">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>180</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Website:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_46">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>240</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Ingresos Brutos:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lne_nombre_fantasia">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>300</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_actualizar">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>510</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Actualizar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
